src/app/shell/personal-cabinet/shared-cabinet/messages/chat/chat.component.ts

```diff
@@ -59,6 +66,7 @@ export class ChatComponent implements OnInit, AfterViewInit, OnDestroy {
   public userIsProvider: boolean;
   public userName: string;
   public companionName: string;
+  public isDisabled$: BehaviorSubject<boolean> = new BehaviorSubject<boolean>(false);
```

Comment:
Do we realy need Subject?  Or can it be just a boolean variable?

Reply:
Can use a simple boolean variable